### DISCUSSION TITLE
Import/Export .csv

### DISCUSSION THREAD

**Discussion by datgausaigon** [6pac/SlickGrid#1010] [Q&A] [answered]
Hi, can I Import/Export .csv with SlickGrid?
Any Docs/Solution/Introduction?
Thanks

**Comment by 6pac** [comment, marked as answer]
Check out SlickGrid-Universal. There are heaps of plugins and enhancements there.

**Reply by datgausaigon** [reply]
@6pac Thanks ❤️
